unison-src/parser-tests/if-without-condition.message.txt

```diff
@@ -0,0 +1,3 @@
+if-without-condition.u:1:10:
+unexpected then
+    1 | [0m
```

Comment:
Since parse errors have ansi codes in them (and we want to see them in color), can you modify your script to print out all the messages to the terminal? Still generate the file too but for the file, let’s emit those as plain text.

@aryairani can you show how to display a parse error sans ANSI codes?